### DISCUSSION TITLE
Required Secure Connections not supported?

### DISCUSSION THREAD

**Discussion by binamov** [Taxel/PlexTraktSync#1257] [Support]
Hi folks,

My plex server has **Secure Connections** set to **Required**.  The certs and all are configured correctly. When I run `plextraktsync` for the first time, I can see the `https://my.serv.er:32400` under `Server Connections`, but then the actual connection attempts happen to `http://my.serv.er:32400` and fail...

I even added `PLEX_BASEURL=https://my.serv.er:32400` to `.env` but this doesn't fix

That doesn't seem to work, or am I missing something?

**Comment by glensc** [comment]
missing version info in the problem report. but the file to be edited is servers.yml since 0.24.0

**Comment by binamov** [comment]
I've kept trying with `latest` docker. It got connecting at some point, but could never sync. I'm now on 0.25.5 and getting `sigkill` from the container.
